apps/angular/projection/src/app/data-access/city.store.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+import { City } from '../model/city.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CityStore {
+  private cities = new BehaviorSubject<City[]>([]);
+  cities$ = this.cities.asObservable();
```

Comment:
you can now use signals 👍

Reply:
Seems like in angular it makes more sense than observables in many cases but React devs seem to think it goes against the data flow of the library (not to mention is not ready for SSR in React, like NextJS), will try it in Angular :)

Reply:
I haven't done a lot of react, but signals are simpler to understand than Rxjs. It will make your app more performant in the future because change detection will be better targeted to the right component.